Pod/Classes/PINRemoteImageManager.m

```diff
@@ -508,6 +508,21 @@ - (NSUUID *)downloadImageWithURL:(NSURL *)url
         }
     }
     
+    if ([url.absoluteString hasPrefix:@"data:"]) {
+        NSData *data = [NSData dataWithContentsOfURL:url];
+        if (data) {
+            id object;
+            if ([url.resourceSpecifier isEqualToString:@"image/gif"]) {
```

Comment:
The resourceSpecifier will contain the data as well. It's everything after "data:"